client/api/src/main/java/org/opennms/velocloud/client/api/model/Edge.java

```diff
@@ -85,6 +154,86 @@ public Builder withHub(final boolean hub) {
             return this;
         }
 
+        public Builder withLinks(final List<Link> links) {
+            this.links = links;
+            return this;
+        }
+
+        public Builder withAlertsEnabled(Boolean alertsEnabled) {
+            this.alertsEnabled = alertsEnabled;
+            return this;
+        }
+
+        public Builder withBuildNumber(String buildNumber) {
+            this.buildNumber = buildNumber;
+            return this;
+        }
+
+        public Builder withCustomInfo(String customInfo) {
+            this.customInfo = customInfo;
+            return this;
+        }
+
+        public Builder withDescription(String description) {
+            this.description = description;
+            return this;
+        }
+
+        public Builder withDeviceFamily(String deviceFamily) {
+            this.deviceFamily = deviceFamily;
+            return this;
+        }
+
+        public Builder withDeviceId(String deviceId) {
+            this.deviceId = deviceId;
+            return this;
+        }
+
+        public Builder withDnsName(String dnsName) {
+            this.dnsName = dnsName;
+            return this;
+        }
+
+        public Builder withLteRegion(String lteRegion) {
+            this.lteRegion = lteRegion;
+            return this;
+        }
+
+        public Builder withLogicalId(String logicalId) {
+            this.logicalId = logicalId;
+            return this;
+        }
+
+        public Builder withModelNumber(String modelNumber) {
+            this.modelNumber = modelNumber;
+            return this;
+        }
+
+        public Builder withName(String name) {
+            this.name = name;
+            return this;
+        }
+
+        public Builder withOperatorAlertsEnabled(Boolean operatorAlertsEnabled) {
+            this.operatorAlertsEnabled = operatorAlertsEnabled;
+            return this;
+        }
+
+        public Builder withSelfMacAddress(String selfMacAddress) {
+            this.selfMacAddress = selfMacAddress;
+            return this;
+        }
+
+        public Builder withSiteId(Integer siteId) {
+            this.siteId = siteId;
+            return this;
+        }
+
+        public Builder withSoftwareVersion(String softwareVersion) {
+            this.softwareVersion = softwareVersion;
+            return this;
+        }
+
```

Comment:
I think we need a toString() method also

Reply:
Do we need one in the builder? The outer class has one on line 85-110

Reply:
Never mind I didn't see at the top.